ui/admin/tests/acceptance/aliases/delete-test.js

```diff
@@ -17,7 +17,8 @@ module('Acceptance | aliases | delete', function (hooks) {
 
   let aliasCount;
 
-  const DROPDOWN_BUTTON_SELECTOR = '.hds-dropdown-toggle-icon';
+  const DROPDOWN_BUTTON_SELECTOR =
+    'tbody tr td:last-child .hds-dropdown-toggle-icon';
```

Comment:
asking for learning: Did this need to be updated since we have more dropdown (specifically in the new side nav) elements on the screen? Either way, I think this is a good change.

Reply:
yup that is correct!